### DISCUSSION TITLE
A test case for hourly time step

### DISCUSSION THREAD

**Discussion by vdungnguyen** [mhm-ufz/mHM#59] [Ideas]
Learning that it is very useful with setup of the two test cases (e.g 'test_domain', 'test_domain_2') using daily timestep data. It would be very much instructive and helpful if the mHM developer team can add a test case which runs on hourly timestep data.

**Comment by rohinikumar82** [comment]
Thank you for your suggestion. We will keep this in mind as we progress forward with developing further test cases.

**Comment by 9202014710** [comment]
According to your statement above, it appears that mHM has the capability to perform hourly simulation. I attempted to conduct some hourly simulations and prepared the corresponding hourly input for testing. However, the output still remains in daily format. I would greatly appreciate any assistance or guidance that can be provided in this matter.

**Comment by pallavhydro** [comment]
Hi @9202014710 

Kindly follow the discussion #76  and continue the conversation there, if needed :)

Cheers
Pallav
